.gitignore

```diff
@@ -5,7 +5,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+#*.jar
```

Comment:
Until that happens, can we have the jar sitting locally (in the project) in lib? Helps me have code that compiles (or nearly does) on the repo.